src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private Integer number;
```

Comment:
Ball number cannot be null, we will learn about wrappers and their usage later on, for now lets stick with the primitives
```suggestion
    private int number;
```